Rasa_Bot/actions/actions_common.py

```diff
@@ -37,3 +37,48 @@ async def run(self, dispatcher, tracker, domain):
         logging.info("no celery error")
 
         return []
+
+
+class SendMetadata(Action):
+    def name(self):
+        return "action_send_metadata"
+
+    async def run(self, dispatcher, tracker, domain):
+        id_file = tracker.get_slot("uploaded_file_id")
+        dispatcher.utter_message(
+            json_message={"text": "image",
+                          "attachmentIds": [id_file]},
+        )
+        return[]
+
+
+class UploadFile(Action):
+    def name(self):
+        return "action_upload_file"
+
+    async def run(self, dispatcher, tracker, domain):
+        client = NicedayClient(NICEDAY_API_ENDPOINT)
+        user_id = int(tracker.current_state()['sender_id'])
+
+        filepath = tracker.get_slot('upload_file_path')
+        with open(filepath, 'rb') as content:
+            file = content.read()
+
+        response = client.upload_file(user_id, 'tst.png', filepath, file)
+        file_id = response['id']
+        logging.info(response)
+        logging.info(file_id)
+
+        return[SlotSet("uploaded_file_id", file_id)]
+
+
+class SetFilePath(Action):
+    def name(self):
+        return "action_set_file_path"
+
+    async def run(self, dispatcher, tracker, domain):
+
+        # This is hardcoded for testing. Needs to be set according to the use case

```

Comment:
```suggestion
        # TODO: This is hardcoded for testing. Needs to be set according to the use case

```

Just to make unfinished threads easier to find later ;)